modules/cryptable.rb

```diff
@@ -0,0 +1,36 @@
+module Cryptable
+
+  def index_to_alphabet
```

Comment:
Changed method name to make more sense and to exemplify that it is the reciprocal of alphabet_to_index